### PR TITLE
Normalize page head and OG data for SEO

### PR DESCRIPTION
**Pull request by adamjarling**
## What does this do?
- Small refactor of how we are rendering consistent values to `<title />`, `<meta name="description" content="..." />`, and Open Graph data.
- Prepends Work and Collection item `title` to page `<title />` tag.
- Updates best practices on applying Open Graph values.
- Implements a const helper to hold values related to page `<head /> `meta` values.
- Adds a Github Action to run unit tests on branch pushes (capture any Amplify build failures early).

image

## How to test
Scroll through the site, and inspect the above elements for consistency, default values, and specific values for dynamic route pages.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll": true,
-    "source.organizeImports": false
+    "source.fixAll": "explicit",
+    "source.organizeImports": "never"
```
Comment (mathewjordan) [inline .vscode/settings.json:4]: I'm not sure if we intentionally started including this file. Should we be still?

File: lib/open-graph.test.ts [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
         "Ima description - The images on this web site are from material in the collections of the University Archives of Northwestern University Libraries, are provided for use by its students, faculty and staff, and by other researchers visiting this site, for research consultation and scholarly purposes only. Further distribution and/or any commercial use of the images from this site is not permitted.",
       "og:image":
         "https://iiif.stack.rdc-staging.library.northwestern.edu/iiif/2/93d75ffe-20d8-48ea-9206-8db9114f2731/full/1200,630/0/default.jpg",
       "og:image:secure_url":
         "https://iiif.stack.rdc-staging.library.northwestern.edu/iiif/2/93d75ffe-20d8-48ea-9206-8db9114f2731/full/1200,630/0/default.jpg",
-      "og:site_name":
-        "Hawking dental products in outdoor market, Cuernavaca, Mexico - Digital Collections - Libraries - Northwestern University",
       "og:title":
-        "Hawking dental products in outdoor market, Cuernavaca, Mexico - Digital Collections - Libraries - Northwestern University",
+        "Hawking dental products in outdoor market, Cuernavaca, Mexico",
```
Comment (mathewjordan) [inline lib/open-graph.test.ts:17]: 👍

**Review by mathewjordan — APPROVED** [review]
Looks great other than that `.vscode` file.

File: .vscode/settings.json (hunk 1 above)
Comment (adamjarling) [reply .vscode/settings.json:4]: Honestly I couldn't remember.  At some point, to keep lengthy imports in files to be organized in a sane way, I think we might have tried to sync our settings.  I'll remove this and if it becomes an annoying issue moving forward we can tackle it.

**Pull request merged**